Lib/urllib/parse.py

```diff
@@ -78,6 +78,9 @@
                 '0123456789'
                 '+-.')
 
+# Unsafe bytes to be removed per WHATWG spec
+_UNSAFE_URL_BYTES_TO_REMOVE = ['\t', '\r', '\n']
```

Comment:
If it is only used in one place and we do not want users to override it, why not inline it?

Reply:
I guess this was done based on my comment : https://github.com/python/cpython/pull/25595#discussion_r622224196 . Similar validation was done in past to check for non-ascii values through regex and cherrypy wanted an option to override. So I just added that comment to be on safe side.

Reply:
Yes, the change was based on the comment.  Although, we shouldn't recommend overriding this, but preventing of override (for any unforeseen practical reason) by in-lining it sounded bit inflexible.

Plus there is also prior-art. In this module, right above this line. And
https://github.com/python/cpython/pull/12755/files
https://github.com/python/cpython/pull/18485/files
https://github.com/python/cpython/commit/a112a8ae47813f75aa8ad27ee8c42a7c2e937d13

Reply:
I'm fine with this (while I'd write it as a tuple if not just '\r\n\t' itself, it won't realistically matter).  The follow-on issue (43883) might lead to further refactoring of this kind of thing FWIW.